packages/zowe-explorer/src/trees/dataset/DatasetActions.ts

```diff
@@ -1079,6 +1079,109 @@ export class DatasetActions {
         }
     }
 
+    /**
+     * Attempts to open a data set from the selection of text made in the editor
+     * Works the same as ZOOM command in TSO/ISPF
+     *
+     */
+    public static async zoom(): Promise<void> {
+        ZoweLogger.trace("dataset.actions.zoom called.");
+
+        const editor = vscode.window.activeTextEditor;
+        if (!editor) {
+            const notActiveEditorMsg = vscode.l10n.t("No active editor open. Please open a file and select text to open a data set.");
+            Gui.warningMessage(notActiveEditorMsg);
```

Comment:
We can pass the value from `vscode.l10n.t` directly to `Gui.warningMessage` and remove the variable declaration as its not used elsewhere:

```suggestion
            Gui.warningMessage(
                vscode.l10n.t("No active editor open. Please open a file and select text to open a data set.")
            );
```